pre-train/reader/extract_features.py

```diff
@@ -0,0 +1,94 @@
+import librosa
+import numpy as np
+import glob
+import os
+from multiprocessing import Pool, cpu_count
+import sys
+    
+def extract_mel_spec(filename):
+    y, sample_rate = librosa.load(filename)
+    mel_spec = librosa.feature.melspectrogram(y=y,
+                                              sr=sample_rate,
+                                              n_fft=2048,
+                                              hop_length=512,
+                                              win_length=None,
+                                              window='hann',
+                                              center=True,
+                                              pad_mode='reflect',
+                                              power=2.0,
+                                              n_mels=128,
+                                              fmin=0.0,
+                                              fmax=None,
+                                              htk=False,
+                                              norm=1)
```

Comment:
To be consistent with other part of code. several defualt parameters shoud be:
hop_length=200,
win_length=800,
power=1.0,
n_mels=80

Reply:
adressed in 997dd6ec9ab34a128189f46c819e03da03b9433e